api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1841,6 +1841,62 @@ message CreateAuthenticateChallengeRequest {
   // If you are issuing challenges from the root cluster, but accessing a leaf,
   // call [AuthService.IsMFARequired] in the leaf instead of setting this field.
   IsMFARequiredRequest MFARequiredCheck = 5 [(gogoproto.jsontag) = "mfa_required_check,omitempty"];
+  // Extensions are extensions that will be apply to the issued MFA challenge.
+  // Extensions only apply to webauthn challenges currently. This field is required.
```

Comment:
> This field is required

Is this true? What about legacy clients?

Reply:
Yeah, I think it won't hurt to say that clients before v15 aren't required to supply the field.